modules/memstore_redis/outputs.tf

```diff
@@ -26,3 +26,13 @@ output "read_endpoint_port" {
   value       = google_redis_instance.cache.read_endpoint_port
   description = "The port of the Redis instance."
 }
+
+output "server_ca_certificates" {
+  value       = google_redis_instance.cache.server_ca_certs
```

Comment:
Can you add a one-liner comment, or description, mentioning the format of the certificates? i.e. is it a binary, a string, encoded, base64? it is plural, so I understand is multiple certificates, is this a list of strings?